libs/application/templates/general-fishing-license/src/fields/Declined/WarningSection.css.ts

```diff
@@ -0,0 +1,52 @@
+import { style } from '@vanilla-extract/css'
+import { theme } from '@island.is/island-ui/theme'
+
+export const sectionNumber = style({
+  borderRadius: '50%',
+  fontWeight: theme.typography.semiBold,
+  height: '32px',
+  left: '-16px',
+  top: 'calc(50% - 16px)',
+  width: '32px',
+})
+
+export const sectionNumberText = style({
+  marginLeft: '-1px',
+})
+
+export const sectionNumberNotStarted = style({
+  background: theme.color.white,
+  border: `1px solid ${theme.color.blue200}`,
+  color: theme.color.blue200,
+})
+
+export const sectionNumberInProgress = style({
+  background: theme.color.blue100,
+  border: `1px solid ${theme.color.blue200}`,
+  color: theme.color.blue400,
+})
+
+export const sectionNumberRequiresAction = style({
+  background: theme.color.red600,
+  color: theme.color.red600,
+})
+
+export const sectionNumberComplete = style({
+  background: theme.color.mint600,
+})
```

Comment:
Most of these can be created using the Box component, and thus this isn't needed.

Reply:
Perfect thank you for this, I'll take a look at this :D. Just copy pasted this from drivers license to get the payment going, I can also see that I'm only using two of these styles!